docs/appendices/release-notes/5.8.3.rst

```diff
@@ -84,3 +84,6 @@ Fixes
 
 - Fixed an issue that caused filtering by ``LIKE ALL`` on arrays to behave like
   ``LIKE ANY``.
+
```

Comment:
just a heads up, since 5.8.3 is currently being released, it will have to be moved to 5.8.4.rst.